src/test/resources/features/highlighting/pythonregexp_highlighting_in_wildcard_constraints_section.feature

```diff
@@ -0,0 +1,16 @@
+Feature: This feature checks reference highlighting in wildcard_constraints sections
+
+  Scenario Outline: Highlight in 'wildcard_constraints' sections
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    <rule_name> NAME:
+      wildcard_constraints:
+          <arg>="<pyredexp>"
+    """
+    When I put the caret after <pyredexp>
+    Then I expect language injection on "<pyredexp>"
```

Comment:
Please also check that proper language is injected, not some random language. You'll need to add/modify the cucumber step definition for this.

Reply:
Done